src/pages/ReviewResults.js

```diff
@@ -64,6 +58,16 @@ export default function ReviewResults(props) {
     setAccessElection(accessElection);
     setAccessElectionReview(accessElectionReview);
     setRpElectionReview(rpElectionReview);
+
+    //get the duos matching algorithm decision
+    let matchData;
+
+    try {
+      matchData = await Match.findMatch(consent.consentId, accessElection.referenceId);
+      setMatchData(matchData);
+    } catch (e) {
+      Notifications.showError({ text: `Something went wrong trying to get match algorithm results. Error code: ${e.status}` });
+    }
   };
```

Comment:
combined two methods so that I could include the retrieval of matchData and have the two arguments I need in the same place